app/helpers/effective_pages_helper.rb

```diff
@@ -13,16 +13,25 @@ def effective_pages_body_classes
   def effective_pages_header_tags
     [
       content_tag(:title, effective_pages_site_title),
-      tag(:meta, name: 'description'.freeze, content: effective_pages_meta_description)
+      tag(:meta, name: 'description'.freeze, content: effective_pages_meta_description),
+      tag(:meta, property: 'og:site_name',   content: EffectivePages.site_title.to_s),
+      tag(:meta, property: 'og:title',       content: effective_pages_page_title),
+      tag(:meta, property: 'og:description', content: effective_pages_meta_description),
+      tag(:meta, property: 'og:url',         content: request.original_url),
+      tag(:meta, property: 'og:image',       content: asset_path(EffectivePages.site_og_image.to_s))
     ].compact.join("\n").html_safe
   end
 
-  def effective_pages_site_title
+  def effective_pages_page_title
```

Comment:
I needed to be able to provide og:title a value that didn't include the site_title_suffix